### PR TITLE
[TD-13201]<test>:test cases for Node.js stmt

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 1]
```diff
@@ -0,0 +1,915 @@
...
+        `(ts timestamp,` +
+        `bl bool,` +
+        `i8 tinyint,` +
+        `i16 smallint,` +
+        `i32 int,` +
+        `i64 bigint,` +
+        `f32 float,` +
+        `d64 double,` +
+        `bnr binary(20),` +
+        `blob nchar(20),` +
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: 3.0 will have a new type blob, we should not use it here to avoid rename it for 3.0.

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 2]
```diff
@@ -0,0 +1,915 @@
...
+let tinyIntArr = [-127, 3, 127, 0, null];
+let smallIntArr = [-32767, 16, 32766, 0, null];
+let intArr = [-2147483647, 17, 2147483646, 0, null];
+let bigIntArr = [16424352000002222n, -16424354000001111n, 18n, 0n, null];
+let floatArr = [3.4028234663852886e+38, -3.4028234663852886e+38, 19, 0, null];
+let doubleArr = [1.7976931348623157e+308, -1.7976931348623157e+308, 20, 0, null];
+let binaryArr = ['TDengine_Binary', 'taosdata涛思数据', '~!@#$%^&*()', '', null];
+let ncharArr = ['TDengine_Nchar', 'taosdata涛思数据', '~!@#$$%^&*()', '', null];
+let uTinyIntArr = [0, 127, 254, 23, null];
+let uSmallIntArr = [0, 256, 512, 24, null];
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: boundary value is not correct

**Review by zitsen — APPROVED** [review]
LGTM

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 3]
```diff
@@ -0,0 +1,915 @@
...
+        `)tags(` +
+        `t_bl bool,` +
+        `t_i8 tinyint,` +
+        `t_i16 smallint,` +
+        `t_i32 int,` +
+        `t_i64 bigint,` +
+        `t_f32 float,` +
+        `t_d64 double,` +
+        `t_bnr binary(20),` +
+        `t_blob nchar(20),` +
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: still blob

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 4]
```diff
@@ -0,0 +1,915 @@
...
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: still blob

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 5]
```diff
@@ -0,0 +1,915 @@
...
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: still blob

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 6]
```diff
@@ -0,0 +1,915 @@
...
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: still blob

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 7]
```diff
@@ -0,0 +1,915 @@
...
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: still blob

File: src/connector/nodejs/test/cases/test.stmt.js [hunk 8]
```diff
@@ -0,0 +1,915 @@
...
```
Comment (sangshuduo) [inline src/connector/nodejs/test/cases/test.stmt.js]: blob

**Review by sangshuduo — APPROVED** [review]
LGTM
